beacon_node/src/config.rs

```diff
@@ -1480,10 +1480,12 @@ pub fn set_network_config(
     if parse_flag(cli_args, "proposer-only") {
         config.subscribe_all_subnets = false;
 
-        if cli_args.get_one::<String>("target-peers").is_none() {
-            // If a custom value is not set, change the default to 15
-            config.target_peers = 15;
-        }
```

Comment:
Why was this here in the first place?

Reply:
I'm going to revert the target peer related changes, as @AgeManning is covering them properly in #7877